ant-protocol/src/storage/pointer.rs

```diff
@@ -16,7 +16,7 @@ use xor_name::XorName;
 #[derive(Clone, Serialize, Deserialize, PartialEq, Eq, Ord, PartialOrd)]
 pub struct Pointer {
     owner: PublicKey,
-    counter: u32,
+    counter: u64,
```

Comment:
deserialize an old Pointer Record with u32 counter into new u64 one will be problematic as well ?